src/routes/OrganizationsList/OrganizationsListTableData.tsx

```diff
@@ -0,0 +1,187 @@
+import {Td} from '@patternfly/react-table';
+import {Skeleton} from '@patternfly/react-core';
+import './css/Organizations.scss';
+import {Link} from 'react-router-dom';
+import {useEffect, useState} from 'react';
+import {fetchOrg} from 'src/resources/OrganizationResource';
+import {
+  fetchRepositoriesForNamespace,
+  IRepository,
+} from 'src/resources/RepositoryResource';
+import {fetchMembersForOrg} from 'src/resources/MembersResource';
+import {fetchRobotsForNamespace} from 'src/resources/RobotsResource';
+import {formatDate} from 'src/libs/utils';
+import ColumnNames from './ColumnNames';
+
+type TeamCountProps = {
+  orgName: string;
+  user: boolean;
+};
+
+type MemberCountProps = {
+  orgName: string;
+  user: boolean;
+};
+
+type RobotCountProps = {
+  orgName: string;
+};
+
+type RepoCountProps = {
+  count: string | number;
+};
+
+type OrgTableProps = {
+  orgName: string;
+  user: boolean;
+};
+
+type RepoLastModifiedDateProps = {
+  lastModifiedDate: number;
+};
+
+function TeamCount(props: TeamCountProps) {
+  const [teamCount, setTeamCount] = useState<string | number>(null);
+
+  useEffect(() => {
+    let count = null;
+    if (!props.user) {
+      fetchOrg(props.orgName)
+        .then((data) => {
+          count = data?.teams ? Object.keys(data?.teams)?.length : 0;
+          setTeamCount(count);
+        })
+        .catch((err) => {
+          console.error(err);
+          count = 'N/A';
+          setTeamCount(count);
+        });
+    } else {
+      count = 'N/A';
+      setTeamCount(count);
```

Comment:
can we convert this to `async` to be consistent across the board? You could use a `try/catch/finally` and set the `setTeamCount(count)` in the `finally` clause